examples/v2beta1/pi/pi.yaml

```diff
@@ -35,6 +30,12 @@ spec:
           containers:
           - image: mpioperator/mpi-pi
             name: mpi-worker
+            lifecycle:
+              postStart:
+            # exec command below makes sure ssh service on port 22 is up and running -- checking localhost but should be
```

Comment:
But what does this prevent?

Were you meant to use a readinessProbe instead? I set up one for the intel scenario https://github.com/kubeflow/mpi-operator/blob/master/examples/v2beta1/pi/pi-intel.yaml